SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/DynamicSubobjectsTest/ReplicatedGASTestActor.cpp

```diff
@@ -5,7 +5,7 @@
 
 AReplicatedGASTestActor::AReplicatedGASTestActor()
 {
-	TestIntProperty = 0;
+	TestIntProperty = -1;
```

Comment:
What's the reason for this?

Reply:
previously, in the main test the server set the int property to the loop variable i+1 - and then the client tested it cannot see that. based on a miron comment i had, i changed it to set the loop variable to just i, but to do so the initial value couldn't be 0 (otherwise the server setting it to 0 does nothing)

Reply:
Aight